Indicators/CompositeIndicator.cs

```diff
@@ -98,13 +98,8 @@ public CompositeIndicator(string name, IndicatorBase<T> left, IndicatorBase<T> r
         /// <param name="right">The right indidcator for the 'composoer'</param>
         /// <param name="composer">Function used to compose the left and right indicators</param>
         public CompositeIndicator(IndicatorBase<T> left, IndicatorBase<T> right, IndicatorComposer composer)
-            : base(string.Format("COMPOSE({0},{1})", left.Name, right.Name))
-        {
-            _composer = composer;
-            Left = left;
-            Right = right;
-            ConfigureEventHandlers();
-        }
+            : this(string.Format("COMPOSE({0},{1})", left.Name, right.Name), left, right, composer)
+        { }
```

Comment:
no action required -- I was always more partial to these being on separate lines.

Reply:
also, I like the removal of the duplication here, good stuff.